src/tests/baseservices/exceptions/exceptioninterop/ExceptionInterop_ro.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
       <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+      <NativeAotIncompatible>true</NativeAotIncompatible>
```

Comment:
I assume we have good coverage for the other exception tests in the tree given the differing exception model used by NativeAOT

Reply:
this test is specifically testing exception interop - like the name says - being able to throw an exception from native code and propagate that through managed code. it's an instant failfast in NativeAOT and we don't have plans to implement it. It can only be implemented on Windows. all test are marked Windows-specific and SkipOnMono because Mono doesn't do this either.